packages/workspace/src/api/WorkspaceConfig.ts

```diff
@@ -1,4 +1,4 @@
-import { ServiceDefinition } from '@scalecube/scalecube-microservice/lib/api';
+import { ServiceDefinition } from '@scalecube/scalecube-microservice/dist/esm/api/public';
```

Comment:
Why do you change this source ?

Reply:
Because of there no this folder, even more there no dependencies, I installed them and tried to build.
Then debugged that there no this folder.